### PR TITLE
Wrap all the fenics stuff in an import alarm

### PR DESCRIPTION
**Pull request by liamhuber**
This is a pretty heavy use of `ImportAlarm` -- since the modules in question get used in a bunch of different places I decided to do the alarming all in a single module and then access the modules via that one alarmed location. For one-off usage it looks more like [this](https://github.com/pyiron/pyiron_continuum/blob/cbc7c08419550ba49275bbc234f6bbe590c8de0f/pyiron_continuum/schroedinger/schroedinger.py#L19).

@samwaseda I don't have pyiron_continuum cloned and set up on the cluster, do you? If so, could you checkout this branch real quick and see if it works as intended on machines that *don't* have fenics?

### CODE REVIEW

File: pyiron_continuum/fenics/import_control.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    "Copyright 2021, Max-Planck-Institut für Eisenforschung GmbH - "
+    "Computational Materials Design (CM) Department"
+)
+__version__ = "0.1"
+__maintainer__ = "Liam Huber"
+__email__ = "huber@mpie.de"
+__status__ = "development"
+__date__ = "Oct 4, 2021"
+
+try:
```
Comment (pmrv) [inline pyiron_continuum/fenics/import_control.py:22]:
```suggestion
with ImportAlarm("Fenics functionality ...") as fenics_alarm:
```
is equivalent.

**Comment by samwaseda** [comment]
```
~/dev_sam/pyiron_continuum/pyiron_continuum/fenics/import_control.py in FEN(self)
     39     @property
     40     def FEN(self):
---> 41         return FEN
     42 
     43     @property

NameError: name 'FEN' is not defined
```

**Comment by liamhuber** [comment]
@samwaseda Ok, please try once more. If this fails too then tomorrow I'll get off my lazy butt and clone continuum onto garching so I can test by myself.

**Comment by samwaseda** [comment]
It's actually the same XD

```python
~/dev_sam/pyiron_continuum/pyiron_continuum/fenics/factory.py in <module>
      9 from pyiron_base import PyironFactory
     10 from pyiron_continuum.fenics.import_control import fm
---> 11 FEN = fm.FEN
     12 mshr = fm.mshr
     13 

~/dev_sam/pyiron_base/pyiron_base/generic/util.py in decorator(*args, **kwargs)
    283         def decorator(*args, **kwargs):
    284             self.warn_if_failed()
--> 285             return function(*args, **kwargs)
    286         return decorator
    287 

~/dev_sam/pyiron_continuum/pyiron_continuum/fenics/import_control.py in FEN(self)
     40     @fenics_alarm
     41     def FEN(self):
---> 42         return FEN
     43 
     44     @property

NameError: name 'FEN' is not defined
```

**Comment by liamhuber** [comment]
*sigh* ok. If it's time sensitive you can just go back to master and apply the `with ... as fenics_alarm` to each module separately. Sometime tomorrow I'll dig into what I'm misunderstanding about the import alarm and hopefully figure out a well-behaved way to centralize the behaviour so we can avoid some code duplication.

**Pull request closed without merging**